.github/workflows/pull-request.yml

```diff
@@ -2,7 +2,7 @@ name: Pull Request
 
 on:
   pull_request:
-    branches: ["**"]
+    branches: ["dev"]
```

Comment:
Question not related with this PR, why snapcraft 7.x/stable is used to build this snap?

Reply:
Because the snap is based on `core18` which is unsupported by 8.x. Sublime Text is proprietary, and seems to be built against library versions that are only available in `core18`, so we're stuck there for now.